setup.cfg

```diff
@@ -0,0 +1,3 @@
+[pycodestyle]
```

Comment:
About `setup.cfg`: is this file really necessary? You also mentioned that you don't like the redundant configuration, and I totally agree, I also don't like the redundancy. I think it would be better to remove setup.cfg. 

Note that running pycodestyle in a stand-alone manner is still possibly using `pylama -l pycodestyle`

Reply:
> pylama -l pycodestyle

I didn't know that.